src/cpp/utils.cpp

```diff
@@ -0,0 +1,33 @@
+#include "utils.h"
+
+#include <fstream>
+#include <iomanip>
+
+namespace effil {
+
+Logger getLogger() {
```

Comment:
I want you to hide all logic about this inside `Logger`.

Reply:
I had replied about that here:
https://github.com/effil/effil/pull/115#discussion_r227430956